app/src/main/kotlin/com/wire/android/ui/home/newconversation/search/ContactSearchResultItem.kt

```diff
@@ -0,0 +1,42 @@
+package com.wire.android.ui.home.newconversation.search
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import com.wire.android.model.UserStatus
+import com.wire.android.ui.common.RowItemTemplate
+import com.wire.android.ui.common.UserProfileAvatar
+import com.wire.android.ui.home.newconversation.contacts.Contact
+
+@Composable
+fun ContactSearchResultItem(
+    contactSearchResult: Contact,
+    searchQuery: String,
+    modifier: Modifier = Modifier
+) {
+    with(contactSearchResult) {
+        RowItemTemplate(
+            leadingIcon = {
+                UserProfileAvatar(
+                    avatarUrl = "",
+                    status = UserStatus.AVAILABLE
+                )
+            },
+            title = {
+                HighLightName(
+                    name = name,
+                    searchQuery = searchQuery
+                )
+            },
+            subTitle = {
+                HighLightSubTitle(
+                    subTitle = label,
+                    searchQuery = searchQuery
+                )
+            },
+            eventType = eventType,
+            onRowItemClicked = {},
+            onRowItemLongClicked = {},
+            modifier = modifier
+        )
+    }
+}
```

Comment:
I see this `ContactSearchResultItem` is almost the same than `ExternalContactResultItem` in shape and behavior except for the fact that the latter one has also an `AddContactButton`. For the sake of DRY prinpiple, do you think there is a way we can merge these 2 into one common element by for example, just passing an `itemType` value that will differentiate them?

Reply:
true,  good idea indeed